framework/auth/oauth_scopes.py

```diff
@@ -124,7 +124,7 @@ class ComposedScopes(object):
     USERS_READ = (CoreScopes.USERS_READ, CoreScopes.ALWAYS_PUBLIC, )
     USERS_WRITE = USERS_READ + (CoreScopes.USERS_WRITE,)
     USERS_CREATE = USERS_READ + (CoreScopes.USERS_CREATE, )
-    USERS_EMAIL = (CoreScopes.USERS_EMAIL, )
+    USER_EMAIL_READ =  (CoreScopes.USERS_READ, CoreScopes.USER_EMAIL_READ, )
```

Comment:
its easier for us to justify adding permissions later if we decide that is the correct case, rather than retracting them and breaking existing integrations...